### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/ACS/acs_reduction/acs_reduction.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#184]
## What
An accessibility scan flagged 22 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/ACS/acs_reduction/acs_reduction.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell2">` (selector: `#codecell2`)
- `<pre id="codecell3"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960a7q_spt.fits to ./mastDownload/HST/j9l960a7q/j9l960a7q_spt.fits ...
</pre>` (selector: `#codecell3`)
- `<pre id="codecell5"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960a7q_raw.fits to ./mastDownload/HST/j9l960a7q/j9l960a7q_raw.fits ...
</pre>` (selector: `#codecell5`)
- `<pre id="codecell7"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960a9q_spt.fits to ./mastDownload/HST/j9l960a9q/j9l960a9q_spt.fits ...
</pre>` (selector: `#codecell7`)
- `<pre id="codecell9"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960a9q_raw.fits to ./mastDownload/HST/j9l960a9q/j9l960a9q_raw.fits ...
</pre>` (selector: `#codecell9`)
- `<pre id="codecell11"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960abq_spt.fits to ./mastDownload/HST/j9l960abq/j9l960abq_spt.fits ...
</pre>` (selector: `#codecell11`)
- `<pre id="codecell13"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960abq_raw.fits to ./mastDownload/HST/j9l960abq/j9l960abq_raw.fits ...
</pre>` (selector: `#codecell13`)
- `<pre id="codecell15"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960adq_spt.fits to ./mastDownload/HST/j9l960adq/j9l960adq_spt.fits ...
</pre>` (selector: `#codecell15`)
- `<pre id="codecell17"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960adq_raw.fits to ./mastDownload/HST/j9l960adq/j9l960adq_raw.fits ...
</pre>` (selector: `#codecell17`)
- `<pre id="codecell19"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960afq_spt.fits to ./mastDownload/HST/j9l960afq/j9l960afq_spt.fits ...
</pre>` (selector: `#codecell19`)
- `<pre id="codecell21"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960afq_raw.fits to ./mastDownload/HST/j9l960afq/j9l960afq_raw.fits ...
</pre>` (selector: `#codecell21`)
- `<pre id="codecell23"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960010_spt.fits to ./mastDownload/HST/j9l960010/j9l960010_spt.fits ...
</pre>` (selector: `#codecell23`)
- `<pre id="codecell25"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j9l960010_asn.fits to ./mastDownload/HST/j9l960010/j9l960010_asn.fits ...
</pre>` (selector: `#codecell25`)
- `<pre id="codecell31">` (selector: `#codecell31`)
- `<pre id="codecell56">` (selector: `#codecell56`)
- `<pre id="codecell57">` (selector: `#codecell57`)
- `<pre id="codecell58">` (selector: `#codecell58`)
- `<pre id="codecell59">` (selector: `#codecell59`)
- `<pre id="codecell66">` (selector: `#codecell66`)
- `<pre id="codecell67">` (selector: `#codecell67`)
- `<pre id="codecell68">` (selector: `#codecell68`)
- `<pre id="codecell69">` (selector: `#codecell69`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
